requirements.txt

```diff
@@ -0,0 +1,7 @@
+requests == 2.10
```

Comment:
I think this file is redundant if we're creating a wheel with setup.py and the requirements are already listed in `setup_requires`. see https://github.com/dataxu/jira-webhook/blob/master/requirements.txt